app/jobs/cache/record_counts/recent_additions_job.rb

```diff
@@ -0,0 +1,48 @@
+module Cache
+  module RecordCounts
+    class RecentAdditionsJob < Cache::RecordCountsJob
+      def perform
+        %w(music art-history).each do |channel|
+          cache_key = "record/counts/channels/#{channel}/recent-additions"
+          channel_params = Channel.find(channel).config[:params]
+          Rails.cache.write(cache_key, recent_additions(channel_params))
+        end
+      end
+
+      protected
+
+      def recent_additions(params = {})
```

Comment:
Assignment Branch Condition size for recent_additions is too high. [29.8/15]<br>Method has too many lines. [23/10]